GDAPI/GDAPI/Application/Editors/Editor.cs

```diff
@@ -485,7 +483,8 @@ public void DeselectObjects(LevelObjectCollection objects, bool registerUndoable
         /// <param name="registerUndoable">Determines whether the events will be invoked. Defaults to <see langword="true"/> and must be set to <see langword="false"/> during undo/redo to avoid endless invocation.</param>
         public void InvertSelection(bool registerUndoable = true)
         {
-            var newObjects = Level.LevelObjects.Clone().RemoveRange(SelectedObjects);
+            var newObjects = Level.LevelObjects.Clone();
+            newObjects.RemoveRange(SelectedObjects);
```

Comment:
is there any reason for this change?

Reply:
This had to happen after changing the implementation of `LevelObjectCollection` into an `IList`, since the interface demands that `RemoveRange`, like other functions return `void`. The consistency should be kept across such functions.